app/views/measures/grouped/_credibility_checks.html.erb

```diff
@@ -0,0 +1,3 @@
+<%# <div class="measure-inset"> %>
```

Comment:
This is just a placeholder for now and we can populate content for it when HMRC have provided it. We do expect them too, soon